src/controllers/employee.js

```
@@ -1,180 +1,606 @@
-exports.showManagers = async (req, res) => {
-  const departmentLeadId = await db.department.findAll({
-    attributes: [["departmentManagerId", "id"]],
+const autoDelete = async () => {
+  let employeeCount = 0;
+  let userCount = 0;
+  console.log("Executing auto delete...");
+  const startTime = dayjs();
+  const employees = await db.employee.findAll({
+    where: { autoDeleteAt: { [db.Sequelize.Op.ne]: null } },
+  });
+  for (let employee of employees) {
```

Biome

[error] This let declares a variable that is only assigned once.

'employee' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)